src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -174,6 +174,25 @@ private ResourceElement CreateBundle(SearchResult result, Bundle.BundleType type
                 });
             }
 
+            if (result.SearchIssues.Any())
+            {
+                // FERNFE - Should I create one operation outcome for each record not matching? Or should I continue doing similar to the BundleIssues?
```

Comment:
@feordin should I create one operation outcome for each record not matching? Or should I continue doing similar to the BundleIssues?

Reply:
I think we should be similar to the BundleIssues, and give one compiled set of issues.